### ISSUE TITLE
Error Login on php7.4 debian

### ISSUE THREAD

**Issue by zickkeen** [asipto/siremis-php#64]
Linux debian 5.10.0-11-amd64
PHP 7.4.25

```
[{"target":"ERROR","content":"
\n[2022-02-26 07:51:46 (GMT)] An exception occurred while executing this script:
\nError message: #8192, Function get_magic_quotes_gpc() is deprecated<\/font>
\nScript name and line number of error: \/var\/www\/siremis\/openbiz\/bin\/ClientProxy.php:190<\/font>
\n

function:<\/b> errorHandler ( 8192, \"Function get_magic_quotes_gpc() is deprecated\", \"\/var\/www\/siremis\/openbiz\/bin\/ClientProxy.php\", 190, Array(3) ) @ \/var\/www\/siremis\/openbiz\/bin\/sysheader.inc 117\n
function:<\/b> userErrorHandler ( 8192, \"Function get_magic_quotes_gpc() is deprecated\", \"\/var\/www\/siremis\/openbiz\/bin\/ClientProxy.php\", 190, Array(3) ) @ \/var\/www\/siremis\/openbiz\/bin\/ClientProxy.php 190\n
function:<\/b> getFormInputs ( \"username\" ) @ \/var\/www\/siremis\/openbiz\/bin\/easy\/EasyForm.php 1401\n
function:<\/b> readInputRecord ( ) @ \/var\/www\/siremis\/siremis\/modules\/user\/form\/LoginForm.php 73\n
function:<\/b> Login ( ) @ \/var\/www\/siremis\/openbiz\/bin\/BizController.php 310\n
function:<\/b> invoke ( ) @ \/var\/www\/siremis\/openbiz\/bin\/BizController.php 110\n
function:<\/b> dispatchRequest ( ) @ \/var\/www\/siremis\/openbiz\/bin\/BizController.php 32\n
function:<\/b> include_once ( \"\/var\/www\/siremis\/openbiz\/bin\/BizController.php\" ) @ \/var\/www\/siremis\/siremis\/bin\/controller.php 6<\/div>\n
Please ask system administrator for help...<\/div>\n"}]
```

**Comment by JoseGoncalves** [comment]
Same issue here... using Debian 11 (PHP 7.4.28).

**Comment by imkermichel** [comment]
same debian 11 php 7.4.28

[{"target":"ERROR","content":"
\n[2022-05-21 23:38:59 (GMT)] An exception occurred while executing this script:
\nError message: #8192, Function get_magic_quotes_gpc() is deprecated<\/font>
\nScript name and line number of error: \/var\/www\/siremis-5.3.x\/openbiz\/bin\/ClientProxy.php:190<\/font>
\n

function:<\/b> errorHandler ( 8192, \"Function get_magic_quotes_gpc() is deprecated\", \"\/var\/www\/siremis-5.3.x\/openbiz\/bin\/ClientProxy.php\", 190, Array(3) ) @ \/var\/www\/siremis-5.3.x\/openbiz\/bin\/sysheader.inc 117\n
function:<\/b> userErrorHandler ( 8192, \"Function get_magic_quotes_gpc() is deprecated\", \"\/var\/www\/siremis-5.3.x\/openbiz\/bin\/ClientProxy.php\", 190, Array(3) ) @ \/var\/www\/siremis-5.3.x\/openbiz\/bin\/ClientProxy.php 190\n
function:<\/b> getFormInputs ( \"username\" ) @ \/var\/www\/siremis-5.3.x\/openbiz\/bin\/easy\/EasyForm.php 1401\n
function:<\/b> readInputRecord ( ) @ \/var\/www\/siremis-5.3.x\/siremis\/modules\/user\/form\/LoginForm.php 73\n
function:<\/b> Login ( ) @ \/var\/www\/siremis-5.3.x\/openbiz\/bin\/BizController.php 310\n
function:<\/b> invoke ( ) @ \/var\/www\/siremis-5.3.x\/openbiz\/bin\/BizController.php 110\n
function:<\/b> dispatchRequest ( ) @ \/var\/www\/siremis-5.3.x\/openbiz\/bin\/BizController.php 32\n
function:<\/b> include_once ( \"\/var\/www\/siremis-5.3.x\/openbiz\/bin\/BizController.php\" ) @ \/var\/www\/siremis-5.3.x\/siremis\/bin\/controller.php 6<\/div>\n
Please ask system administrator for help...<\/div>\n"}]

**Comment by miconda** [comment]
You have to use the git master branch, the issue should be fixed by the commit 6af4ccfaadf08972c3bd701c73fab7f831f081a8 .
